JPS_Ontology/ontology/ontoassetmanagement/OntoAssetManagement.csv

```diff
@@ -30,8 +30,38 @@ UnitPrice,Class,Price,IS-A,,,,Price per individual product or service.,https://w
+MaintenanceSchedule,Class,,,,,,"A maintenance schedule tells and records when an asset is scheduled for maintenance, when was it last maintained, who performs the maintenance etc.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Maintenance schedule
+MaintenanceTask,Class,,,,,,"A small, identifiable piece of work that indicates the actions one must take to complete a maintenance work order.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Maintenance task
+Event,Class,,,,,,"A thing that happens or takes place, especially one of importance.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Event
+AssetUpdateHistory,Class,,,,,,A record of updates applied to the information of an asset.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Asset update history
+Furniture,Class,,,,,,"Objects intended to support various human activities such as seating (e.g., stools, chairs, and sofas ), eating ( tables ), storing items, working, and sleeping (e.g., beds and hammocks ). ",https://www.theworldavatar.com/kg/ontoassetmanagement/,Furniture
+Cabinet,Class,Furniture,IS-A,,,,A case or cupboard with shelves and/or drawers for storing or displaying items.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Cabinet
+Project,Class,,,,,,"Any undertaking, carried out individually or collaboratively and possibly involving research or design, that is carefully planned to achieve a particular goal.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Project
+ResearchGrant,Class,,,,,, A grant obtained for conducting scientific research generally through a competitive process.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Research grant
+Budget,Class,,,,,,The amount of money needed or available for a purpose. ,https://www.theworldavatar.com/kg/ontoassetmanagement/,Budget
+BudgetCategory,Class,,,,,,A sub-divison of a budget allocated for a more specific purpose.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Budget category
+ServiceCode,Class,,,,,,An attribute that describes the service code of a budget category.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Service code
+ServiceCategory,Class,,,,,,An attribute that describes the service category of a budget category.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Service category
+lastAssignedTo,Object Property,,,https://w3id.org/bot#Element,https://spec.edmcouncil.org/fibo/ontology/FND/Parties/Parties/IndependentParty,,A relationship which describes who the asset was last assigned to.,https://www.theworldavatar.com/kg/ontoassetmanagement/,last assigned to
+wasExecutedBy,Object Property,,,Event,https://spec.edmcouncil.org/fibo/ontology/FND/Parties/Parties/IndependentParty,,A relationship which describes the party that executed the event.,https://www.theworldavatar.com/kg/ontoassetmanagement/,was executed by
```

Comment:
Looking at it again, doesn't really convince me. Events can also just happen, without any active party involved (at least according to the parties which we look at, e.g. earth quakes, floods, etc.).
Maybe we shall rename this relationship, or potentially better: find a more specific name for Event, as we do not reuse any external concept for that anyway

Reply:
will replacing Event with AssetUpdateEvent be better?

Reply:
Sounds kinda odd to me. Wouldn't `AssetUpdate` or `AssetDataUpdate` suffice?

Reply:
So instead of AssetUpdateHistory hasEvent Event, it will be something like AssetUpdateHistory hasAssetUpdate AssetDataUpdate?